### PR TITLE
Issue #20 - Develop curve classes

### PR DESCRIPTION
**Pull request by JackBuck**
Added SVGPath curves.

*Note: The changes have already been merged to the milestone branch. Thus this pull request is from `M_MotorFunctionality` to `master`*

### CODE REVIEW

File: roboplot/svg/svg_parsing.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    Returns:
+        an iterable of SVGPath objects.
+
+    """
+    paths, _, svg_attributes = svg.svg2paths2(filepath)
+    setsofsubpaths = [p.continuous_subpaths() for p in paths]
+    paths = itertools.chain.from_iterable(setsofsubpaths)
+
+    svg_attributes = SvgAttributes(svg_attributes)
```
Comment (JackBuck) [inline roboplot/svg/svg_parsing.py]: Fixed :)
But I won't merge the pull request yet since I noticed you've branched off M_MotorFunctionality

**Comment by JackBuck** [comment]
I am currently working on the problems with the last three tests

**Comment by JackBuck** [comment]
I believe this is ready to be merged now @Hsenrab ?

**Comment by Hsenrab** [comment]
Just installed scipy and reran the tests it is quite a difference!

**Comment by Hsenrab** [comment]
Ready to be merged as all tests now pass

**Pull request merged**